docs/user/authoring-input-modes.rst

```diff
@@ -0,0 +1,69 @@
+Authoring Input Modes
+=====================
+
+# Overview
+
+A number of components provide multiple authoring modes.
+
+As a concrete example, the waves generated by the ShapeFFT component can be added to the world in multiple ways, including:
+
+* **Global** - waves simply added everywhere in the world
+* **Painted** - waves are painted down by hand in particular areas
+* **Spline** - waves are added along a user defined spline
+
+These are selected using the **Input Mode** dropdown on the ShapeFFT component, which display the valid modes for each component.
+
+The following sections describe each mode.
+
+
+# Global Mode
+
+The input simply applies everywhere.
+
+
+# Painted Mode
+
+The painting modality allows the input to be directly hand authored into the world. This is flexible and powerful. Keep in mind however that this data may need modification or re-authoring if the environment changes. It would be advisable to add quick input until the environment is locked down.
```

Comment:
"add quick input until" a little confusing.

Reply:
yeah agreed, actually this sentence was unnecessary, removed